core/domain/rte_component_registry_test.py

```diff
@@ -67,23 +67,27 @@ def _validate_customization_arg_specs(self, customization_arg_specs):
             self.assertTrue(isinstance(ca_spec['description'], basestring))
             self.assertGreater(len(ca_spec['description']), 0)
 
-            # The default value might not pass validation checks (e.g. the
-            # Image component has a required field whose default value is
-            # empty). Thus, when checking the default value schema, we don't
-            # apply the custom validators.
             schema_utils_test.validate_schema(ca_spec['schema'])
-            self.assertEqual(
-                ca_spec['default_value'],
-                schema_utils.normalize_against_schema(
-                    ca_spec['default_value'], ca_spec['schema'],
-                    apply_custom_validators=False))
-
-            if ca_spec['schema']['type'] == 'custom':
-                obj_class = obj_services.Registry.get_object_class_by_type(
-                    ca_spec['schema']['obj_type'])
+            # Default value of 'url' customization arg in rich-text component
+            # Link cannot be normalized as it is empty value. Hence when
+            # normalizing default values, we skip url.
+            if ca_spec['name'] != 'url':
```

Comment:
Add an else clause which asserts that the default value for a link is empty.

Reply:
Done.